Script/AtomicNET/AtomicNET/Resource/ResourceCache.cs

```diff
@@ -29,6 +30,20 @@ public System.IO.Stream GetFileStream(string name, bool sendEventOnFailure = tru
             return null;
         }
 
+        public List<string> GetResourceDirs()
+        {
+            List<string> resourceDirs = new List<string>();
+
+            ResourceCache cache = AtomicNET.GetSubsystem<ResourceCache>();
+
```

Comment:
When descended from Object, there is no need to prefix this with ```AtomicNET```, the static ```AtomicNET.GetSubsystem``` is useful when in external code.